trustee-on-azure/01-infrastructure-setup-Azure/01-creating-custom-rhcos-image.md

```diff
@@ -0,0 +1,160 @@
+# Create Customized RHCOS image with 'trustee-attester' + 'aa-client-service' in Initrd
+
+**Authors:** demeng@redhat.com, hhei@redhat.com \
+**Jira Issue:** [VIRTWINKVM-1188](https://issues.redhat.com/browse/VIRTWINKVM-1188)  
+**References:**
+- [Custom CoreOS Disk Images](https://github.com/coreos/custom-coreos-disk-images)
+- [Quay.io Repository](https://quay.io/repository/)
+- [Latest RHCOS Images](https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/latest/)
+
+(Explanatory statement: Include trustee-attester binary file + .sh/.service files in the initrd to ensure the passphrase from the KBS-server can be fetched at the initramfs stage.).
+
+
+### Step 1: Get authentication credentials
+```bashV
+Create ~/.config/containers/auth.json by clicking https://console.redhat.com/openshift/create/local, then copy or download secret.
+```
+
+### Step 2. Deploy fedora41 environment
+```bash
+sudo dnf update -y
+sudo setenforce 0
+sudo sed -i -e 's/SELINUX=enforcing/SELINUX=permissive/' /etc/selinux/config
+sudo dnf install -y osbuild osbuild-tools \
+         osbuild-ostree podman jq xfsprogs \
+         e2fsprogs dosfstools genisoimage \
+         squashfs-tools erofs-utils syslinux-nonlinux \
+mkdir ~/test/create_cos_image; cd ~/test/create_cos_image
+# copy the trustee-attester binary file to this path.
+```
+
+
+### Step 3. Get the relevant code
+This repo contains several scripts that used to build container images for RHCOS.
+```bash
+git clone https://gitlab.com/6-dehan/initrd-trustee_plus.git
```

Comment:
Can you copy the entire content of this repo here in a folder as well so that we have everything in the same place? Thanks

Reply:
Archive a new folder as 'initrd-trustee_plus'.

Reply:
Great, we can now remove this reference and use the content from here